src/features/applicationList/ApplicationList.tsx

```diff
@@ -142,6 +154,32 @@ const ApplicationList = ({
             />
           );
         }}
+        renderTableToolsTop={({ selectedRows }, { resetSelectedRows }) => {
+          const offers = getDraftedOffers(selectedRows);
+          const offersWithoutPlacesCount = selectedRows.filter((row) => !row.lease).length;
+
+          let toastId;
+          if (offersWithoutPlacesCount > 0)
+            toastId = hdsToast({
+              type: 'error',
+              toastId: 'multiApplicationsError',
+              labelText: 'applicationList.errors.unhandledApplications.label',
+              text: 'applicationList.errors.unhandledApplications.description',
+              translated: true,
+            });
```

Comment:
Worked nicely, but I couldn't read it in time 😄 Maybe disable auto-dismiss for this one, with the prop `autoDismiss: false`